spec/plurimath/asciimath/parser_spec.rb

```diff
@@ -1141,5 +1141,103 @@
         expect(formula).to eq(expected_value)
       end
     end
+
+    context "when contains simple decimal" do
+      let(:string) { "0.1" }
+
+      it "returns formula" do
+        expected_value = Plurimath::Math::Formula.new([
+          Plurimath::Math::Number.new("0.1")
+        ])
+        expect(formula).to eq(expected_value)
+      end
+    end
+
+    context "when contains comma sepearted numeric value" do
+      let(:string) { "0,1" }
+
+      it "returns formula" do
+        expected_value = Plurimath::Math::Formula.new([
+          Plurimath::Math::Number.new("0"),
+          Plurimath::Math::Symbol.new(","),
+          Plurimath::Math::Number.new("1"),
```

Comment:
Actually, the original request is to parse these values all identically, i.e.:
* `0,1` == `0.1`
* `-.1` == `-0.1`
* `-0,1` == `-0.1`
* `-.1` == `-0.1`

Therefore the current expectations all need to be updated accordingly.